frontend/src/pages/Report.tsx

```diff
@@ -26,6 +26,7 @@ import {
   getFullWeek,
   dateFormat,
```

Comment:
Change the date format of the dates:
![image](https://user-images.githubusercontent.com/105041852/186682049-1f35e4d6-7c9b-42e9-8af0-fdf4e435b3d0.png)
Use the Swedish standard: ”2013-12-13” or ”13.12.2013”